webpack/loaders/manifest-loader.ts

```diff
@@ -1,11 +1,39 @@
 import fs from 'fs';
+import path from 'path';
 
 import webpack from 'webpack';
 
+// This is duplicated from util/manifest.ts
+// For some reason webpack throws a `require() of ES modules is not supported.` error
+// when trying to import? :shrug:
```

Comment:
Yeah, I wasn't able to fix this either.  This feels like a lot of the issues we had with mocha / etc in the past.  Good enough for now.